### DISCUSSION TITLE
Add a counter to the notification indicator

### DISCUSSION THREAD

**Discussion by mahdyar** [community/community#4988] [Other Feature Feedback, Questions, & Ideas]
Wouldn't this be nicer to add a counter to the notification indicator? :)

![image](https://user-images.githubusercontent.com/20593549/128831200-e890fa0e-33e7-43f7-86c5-7f465f7f7c8d.png)

----

**Please up vote instead of reacting:**
![upvote](https://user-images.githubusercontent.com/20593549/128859736-e5d54bb4-494e-4300-b9c4-26bd6fcf40ee.jpg)

**Comment by Haroenv** [comment]
🔔 (9k)

**Comment by mahdyar** [comment]
Or maybe add it to the tooltip:

<img width="359" alt="image" src="https://user-images.githubusercontent.com/20593549/168478759-ab19e830-0c22-4a57-9685-5c92557fb8e1.png">
